PurchasesCoreSwift/Purchasing/ProductsManager.swift

```diff
@@ -121,4 +123,5 @@ private extension ProductsManager {
 
         cachedProductsByIdentifier.merge(productsByIdentifier) { (_, new) in new }
```

Comment:
Oops, I think the merge could use [the extension that we created](https://github.com/RevenueCat/purchases-ios/pull/716) to reduce the verbosity. The code will look like this:
```swift
cachedProductsByIdentifier.merging(productsByIdentifier)
```

WDYT? 🤔